server/chat-commands/moderation.js

```diff
@@ -1396,7 +1396,9 @@ exports.commands = {
 		this.splitTarget(target);
```

Comment:
Grab the line count from here.

```suggestion
		const lineCount = parseInt(this.splitTarget(target)) || 0;
```